deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -629,6 +626,9 @@ private void ingestWork(ContentContainerObject parent, Resource parentResc, Reso
 
                 overrideModifiedTimestamp(obj, childResc);
 
+                // Increment the count of objects deposited after adding children
+                addClicks(1);
```

Comment:
We probably need to do this for other object types too? Would it make sense to do this at the same time `incrIngestedObjects` happens below?